Fix pylint 2.7 dependencies:
- Remove references to configparser (this will be automatically handled
  via the six module).

These changes will not be sent upstream.

--- pylint-1.6.4/pylint/__pkginfo__.py
+++ pylint-1.6.4/pylint/__pkginfo__.py
@@ -24,7 +24,7 @@ install_requires = [
 
 extras_require = {}
 extras_require[':sys_platform=="win32"'] = ['colorama']
-extras_require[':python_version=="2.7"'] = ['configparser', 'backports.functools_lru_cache']
+extras_require[':python_version=="2.7"'] = ['backports.functools_lru_cache']
 
 
 license = 'GPL'
--- pylint-1.6.4/pylint.egg-info/requires.txt
+++ pylint-1.6.4/pylint.egg-info/requires.txt
@@ -4,7 +4,6 @@ isort >= 4.2.5
 mccabe
 
 [:python_version=="2.7"]
-configparser
 backports.functools_lru_cache
 
 [:sys_platform=="win32"]
